src/components/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from "react";
+import "./Searchbar.css"
+
+class Searchbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchValue: ''
+    }
+  }
+
+  handleInput = (e) => {
+    this.setState({ [e.target.name]: e.target.value })
+    this.props.updateSearch(e.target.value)
+  }
+
+  componentWillUnmount() {
+    this.props.updateSearch('')
+  }
+
+  render() {
+    return (
+      <form className="search">
```

Comment:
I think it's the right move to remove the button even though it looked great!  Nice work.